### PR TITLE
boards: cy8ckit_062_wifi_bt_m{0,4}: Add OpenOCD support for flashing

### PR DESCRIPTION
**Pull request by galak**
The 0.9.5 SDK has support for the Cypress PSoC6 family.  Add ablilty to
flash with openocd.  I've only tested flashing hello_world on the m0.

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

### CODE REVIEW

**Comment by galak** [comment]
@OlegKapshii please take a look

**Comment by pfalcon** [comment]
@OlegKapshii : Cool, thanks for testing, can you approve this PR then?

**Comment by OlegKapshii** [comment]
@pfalcon Can I do this? I do not see any suitable control on the page.

**Comment by pfalcon** [comment]
@OlegKapshii :

> @pfalcon Can I do this? I do not see any suitable control on the page.

Yep, it's a bit hidden for people who aren't in reviewers already. But anyone should be able to cast a vote by clicking on "Files changed" tab, then you'll see "Review changes" green button below the tab headers.

**Review by OlegKapshii — APPROVED** [review]

**Review by pfalcon — APPROVED** [review]
Looks good, and approving based on @OlegKapshii's testing.

File: boards/arm/cy8ckit_062_wifi_bt_m0/board.cmake [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 #
+set_ifndef(BOARD_FLASH_RUNNER openocd)
```
Comment (MaureenHelm) [inline boards/arm/cy8ckit_062_wifi_bt_m0/board.cmake]: This line isn't necessary

File: boards/arm/cy8ckit_062_wifi_bt_m0/board.cmake [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
 
+include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
 include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
-
-set(CY8C6247BZI_D54)
```
Comment (MaureenHelm) [inline boards/arm/cy8ckit_062_wifi_bt_m0/board.cmake]: Is this an unrelated change?

File: boards/arm/cy8ckit_062_wifi_bt_m4/board.cmake [hunk 3]
```diff
@@ -3,5 +3,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 #
+set_ifndef(BOARD_FLASH_RUNNER openocd)
```
Comment (MaureenHelm) [inline boards/arm/cy8ckit_062_wifi_bt_m4/board.cmake]: This line isn't necessary

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

File: boards/arm/cy8ckit_062_wifi_bt_m0/board.cmake (hunk 1 above)
Comment (galak) [reply boards/arm/cy8ckit_062_wifi_bt_m0/board.cmake]: removed

**Comment by galak** [comment]
@MaureenHelm updated w/changes requested, can you re-review.

**Review by MaureenHelm — APPROVED** [review]

**Comment by MaureenHelm** [comment]
recheck
